components/BaseModal.vue

```diff
@@ -5,39 +5,39 @@ const emit = defineEmits<{
   (e: "closeModal"): void;
 }>();
 
-const focusableNodes = computed(() => {
-  return root.value?.querySelectorAll<HTMLInputElement>(
-    "a[href]:not([disabled]), button:not([disabled]), textarea:not([disabled]), input[type='text']:not([disabled]), input[type='radio']:not([disabled]), input[type='checkbox']:not([disabled]), select:not([disabled])"
-  );
-});
+const focusableNodes = () => {
+  if (root.value) {
+    return root.value?.querySelectorAll<HTMLElement>(
+      "a[href]:not([disabled]), button:not([disabled]), textarea:not([disabled]), input[type='text']:not([disabled]), input[type='radio']:not([disabled]), input[type='checkbox']:not([disabled]), select:not([disabled])"
+    );
+  }
+};
 
```

Comment:
I made changes cuz focusableNodes as computed didn't react when the button changed from disabled to enabled, the computed stayed the same. I changed it to a function so that it fires on every tab checking what are to focusable elements.